src/components/combobox/combobox.stories.ts

```diff
@@ -274,6 +279,45 @@ export const darkThemeRTL_TestOnly = (): string => html`
 `;
 darkThemeRTL_TestOnly.parameters = { themes: themesDarkDefault };
 
+export const FlipPositioning = stepStory(
```

Comment:
Let's drop this one since it was removed in the [storybook cleanup PR](https://github.com/Esri/calcite-components/pull/5257/files#diff-bc2137321903a9338f046ec0516d2cd615e61bad7075e894432746133fe121a5).